RecoLocalFastTime/FTLClusterizer/src/MTDThresholdClusterizer.cc

```diff
@@ -303,5 +324,19 @@ FTLCluster MTDThresholdClusterizer::make_cluster(const FTLCluster::FTLHitPos& hi
                      acluster.y.data(),
                      acluster.xmin,
                      acluster.ymin);
+
+  // For BTL compute the optimal position along crystal and uncertainty on it in absolute length units
+
+  if (seed_subdet == GeomDetEnumerators::barrel) {
+    float sumW(0.f), sumXW(0.f), sumXW2(0.f);
+    for (unsigned int index = 0; index < acluster.top(); index++) {
+      sumW += acluster.energy[index];
+      sumXW += acluster.energy[index] * pixel_x[index];
+      sumXW2 += acluster.energy[index] * acluster.energy[index] * pixel_errx2[index];
+    }
+    cluster.setClusterPosX(sumXW / sumW);
+    cluster.setClusterErrorX(std::sqrt(sumXW2 / sumW / sumW));
```

Comment:
```suggestion
    cluster.setClusterErrorX(std::sqrt(sumXW2) / sumW );
```

Reply:
yes, of course, thanks for noticing it...